content/rdi/monitoring-guide.md

```diff
@@ -5,103 +5,68 @@ description: Monitor RDI Engine and data processing jobs
+> Note: The actual value of the metric above can be 0, if you haven't started RDI Engine yet (in which case, the `state` label should indicate that as well). You must have the RDI database created and configured before observing any metrics. If you are not seeing it or getting an error value instead, this indicates that the RDI database is not properly configured.
 
 ## Configure Prometheus
 
 Next, configure the Prometheus scraper. Edit the `prometheus.yml` file to add the following scraper config:
 
 ```yaml
 scrape_configs:
-  # scrape OSS Metrics Exporter
-  - job_name: redis-exporter
+  # scrape RDI metrics exporter
+  - job_name: rdi-exporter
     static_configs:
       - targets: ["redis-exporter:9121"]
-    metric_relabel_configs:
-      - source_labels: [key]
-        regex: ".+operation=([^,}]+).+"
-        target_label: "operation"
-        replacement: "${1}"
-      - source_labels: [key]
-        regex: ".+data_source=([^,]+).+"
-        target_label: "data_source"
-        replacement: "${1}"
-      - source_labels: [key]
-        regex: ".+status=([^,]+).+"
-        target_label: "status"
-        replacement: "${1}"
-      - source_labels: [key]
-        regex: "([^{]+){.+"
-        target_label: "metric_name"
-        replacement: "${1}"
-      - source_labels: [__name__, metric_name]
-        regex: "redis_script_values;(.*)"
-        target_label: __name__
-        replacement: "${1}"
-      - action: labeldrop
-        regex: "metric_name|key"
 ```
 
-> Note: Make sure the `targets` value above points to the host and port you configured to run the OSS Metrics Exporter.
+> Note: Make sure the `targets` value above points to the host and port you configured to run the RDI metrics exporter.
+> Note: The scrape_interval setting in Prometheus should be the same or more than the collect_interval setting for the exporter. For example, if the collect_interval is set to 5 seconds, the scrape_interval should also be set to 5 seconds or more. If the scrape_interval is set to less than the collect_interval, Prometheus will scrape the exporter before it has a chance to collect and refresh metrics, and you will see the same values duplicated in Prometheus.
 
 ### Test Prometheus scraper
 
-After the scraper config is added to the Prometheus configuration, you should now be able to navigate to `http://<HOSTNAME>:9090/graph` (replace `<HOSTNAME>` with a valid Prometheus hostname or IP address). 
-Explore RDI metrics using the [expression browser](https://prometheus.io/docs/visualization/browser/). 
+After the scraper config is added to the Prometheus configuration, you should now be able to navigate to `http://<HOSTNAME>:9090/graph` (replace `<HOSTNAME>` with a valid Prometheus hostname or IP address).
+Explore RDI metrics using the [expression browser](https://prometheus.io/docs/visualization/browser/).
```

Comment:
consider add an empty line before to have them in two different lines.

Reply:
added